index.html

```diff
@@ -58,135 +78,138 @@ <h2>I’m a software developer</h3>
+                </li>
+                <li class="project pr-4 bg-img btn-desktop">
+                    <div class="body-content">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="btn">
+                        <button>See Project</button>
+                    </div>
+                </li>
+                <li class="project pr-5 bg-img btn-desktop">
+                    <div class="body-content">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="btn">
+                        <button>See Project</button>
+                    </div>
+                </li>
+                <li class="project pr-6 bg-img btn-desktop">
+                    <div class="body-content">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="btn">
+                        <button>See Project</button>
+                    </div>
+                </li>
+                <li class="project pr-7 bg-img btn-desktop">
+                    <div class="body-content">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <div class="btn">
+                        <button>See Project</button>
+                    </div>
```

Comment:
- [ ] same thing here kindly make sure all the `<button>See Project</button>` buttons are wrapped in an anchor tag , kindly fix them 🙏